### PR TITLE
IDEX-3681: Show logs of dependency updater to the output console

### PR DESCRIPTION
**Pull request by dimasnurenko**
@vparfonov

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/dependenciesupdater/ChannelParameters.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ *******************************************************************************/
+package org.eclipse.che.ide.ext.java.client.dependenciesupdater;
+
+import org.eclipse.che.ide.extension.machine.client.outputspanel.console.DefaultOutputConsole;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * @author Dmitry Shnurenko
```
Comment (azatsarynnyy) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/dependenciesupdater/ChannelParameters.java]: What about javadoc

Comment (vparfonov) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/dependenciesupdater/ChannelParameters.java]: +1 Hard to understand why we need this class

**Comment by vparfonov** [comment]
@skabashnyuk can you review MavenClassPathBuilder

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/dependenciesupdater/LogsOutputHandler.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import org.eclipse.che.ide.websocket.events.MessageHandler;
+import org.eclipse.che.ide.websocket.rest.StringUnmarshallerWS;
+import org.eclipse.che.ide.websocket.rest.SubscriptionHandler;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/dependenciesupdater/LogsOutputHandler.java]: javadoc

File: plugin-sdk/che-plugin-sdk-env-local/src/main/java/org/eclipse/che/env/local/server/SingleEnvironmentFilter.java [hunk 3]
```diff
@@ -108,7 +111,7 @@ public String getRemoteUser() {
 
             @Override
             public boolean isUserInRole(String role) {
-                return user.isMemberOf(role);
+                return true;
             }
```
Comment (skabashnyuk) [inline plugin-sdk/che-plugin-sdk-env-local/src/main/java/org/eclipse/che/env/local/server/SingleEnvironmentFilter.java]: what for?

**Pull request merged**